executor/aggregate.go

```diff
@@ -455,14 +478,19 @@ func (w *HashAggFinalWorker) consumeIntermData(sctx sessionctx.Context) (err err
 		// Consume input in batches, size of every batch is less than w.maxChunkSize.
 		for reachEnd := false; !reachEnd; {
 			intermDataBuffer, groupKeys, reachEnd = input.getPartialResultBatch(sc, intermDataBuffer[:0], w.aggFuncs, w.maxChunkSize)
```

Comment:
Can we make `getPartialResultBatch` return `[][]byte` ? I'm not sure will it be better.